src/templates/player/player-q-results.twig

```diff
@@ -27,20 +24,221 @@
-            <div class="row my-2 align-items-center">
-                <div class="col">
-                    <b>Word Count:</b> <span id="wordCount">0</span>
-                </div>
-                <div class="col-3">
-                    <a class="btn btn-secondary" href="/api/athlete/{{ player.id }}/report" target="_blank" >View Report</a>
-                </div>
-            </div>
         </form>
+        <div class="d-flex justify-content-between mt-3">
+            <div>
+                <b
+                    style="
+                        font-family: 'Inter', 'Arial', sans-serif;
+                        font-weight: 600;
+                        font-size: 14px;
+                        color: #000000;
+                    "
+                >
+                    Word Count:
+                </b>
+                <span
+                    id="wordCount"
+                    style="
+                        font-family: 'Inter', 'Arial', sans-serif;
+                        font-weight: 400;
+                        font-size: 14px;
+                        color: #000000;
+                    "
+                >
+                    0
+                </span>
+            </div>
+            <a
+             href="/api/athletes/{{ player.id }}/report"
+             target="_blank"
+             class="p-1"
+             style="
+                color: #000000;
+                background-color: #CFFF80;
+                text-align: center;
+                text-decoration: none;
+                font-family: 'Inter', 'Arial', sans-serif;
+                border-radius: 5px;
+                font-weight: 600;
+                font-size: 14px;
+                width: 140px;
+                height: 30px;
+             "
+            >
+                View Full Report
+            </a>
+        </div>
     </div>
     <div class="col">
-        <div id="player-report" class="pt-1 pb-3 px-1 result-tab">{{ results.player }}</div>
-        <div id="coach-report" class="pt-1 pb-3 px-1 result-tab">{{ results.coach }}</div>
-        <div id="scout-report" class="pt-1 pb-3 px-1 result-tab">{{ results.scout }}</div>
+        <div id="player-report" class="pt-1 pb-3 px-1 result-tab">
+            <div class="d-flex justify-content-between">
+                <h5
+                    style="
+                        font-family: 'Inter', 'Arial', sans-serif;
+                        font-weight: 700;
+                        font-size: 24px;
+                        color: #000000;
+                    "
+                >
+                    Player Results
+                </h5>
+                <div class="row justify-content-between">
+                    <a
+                        href="{{ url }}/player/{{ player.candidate_id }}/pdf/report/player/download"
+                        class="p-1"
+                        style="
+                            color: #CFFF80;
```

Comment:
Why aren't you using the Bootstrap button classes? Will save a lot of headaches.